### ISSUE TITLE
Convert TWEEN to Twixt

### ISSUE THREAD

**Issue by samreid** [phetsims/fraction-comparison#39]
As discussed in https://github.com/phetsims/joist/issues/387#issuecomment-419600722, it would be nice to use Twixt instead of TWEEN.  This sim has 2 occurrences of `new TWEEN.Tween(...)`.  Assigning to the responsible dev.  After completing this issue, check if the hold on https://github.com/phetsims/joist/issues/387 can be removed (if this was the last "Convert from TWEEN to Twixt" issue).

**Comment by samreid** [comment]
When I created this issue, I incorrectly guessed that Fraction Comparison was part of the new in-development Fractions suite.  I checked my guess by checking the recent commits, and noted @jonathanolson was the most popular name in the first page of results, so I listed him as responsible-dev and assigned this issue to him.  However, @jonathanolson pointed out this is incorrect.  This sim was initially developed by me starting around 5 years ago, so I'll self-assign.  My apologies for the mistake and confusion!
